### ISSUE TITLE
Star Wars Rogue Squadron 2 - Rogue Leader - US [GSWE64] Second Mission Crash on WiiU

### ISSUE THREAD

**Issue by xXS4ALLx** [FIX94/Nintendont#551]
Star Wars Rogue Squadron 2 - Rogue Leader - US [GSWE64]

As far as testing goes, the USA version of this game crashes when you try to start the second mission on the WiiU, either injected or running straight from SD CARD.
I can confirm my USA version of the ISO is not corrupted as on the original Wii, it works fine, and also on Dolphin emulator, both, the second mission works fine.

I got a copy of the EUR/PAL version of this game and the second mission works fine on the WiiU.

I think most would prefer the USA version because it has progressive scan option, whereas EUR/PAL as PAL60.

Not sure if this can be fixed or not, but just putting this out there.

**Comment by carnage702** [comment]
you sure you dont have diferent settings aplyed or a diferent dol? i played both usa rogue squadron games on vwii fully. i will try this tonight but you could take a picture of your nintendont settings if you can on wiiu.

**Comment by xXS4ALLx** [comment]
Nintendont version I'm using at the moment is:

**Nintendont Loader v5.481 (Wii U)
Built: Jan 23 2018 19:57:51
Firmware: 58.25.32**

During my troubleshooting, I've deleted the **nincfg.bin** file, on the SD CARD and started fresh.
So whatever the Nintendont settings is by default is what I have tried.

I have also just tested by just booting straight into vWii mode, launching Nintendont from the Homebrew Channel and the game still crashes on the second stage.

I also used the same SD CARD with the same US version and inserted on my original soft modded Wii and the second mission works fine.

I have checked the CRC for the US version which is: e9d4823d
According to: http://www.gametdb.com/Wii/GSWE64
Is the correct CRC.

**UPDATE:**
I have started from scratch again (this is like my 5th time), reformatted the SD CARD again and copied a fresh copy of everything, just the main files on the SD CARD. Now it's working!
Don't ask me how it's working, I've done this 5 times already, all those time not working, however this time its working??? I can't explain it if I could.

Well, I can say case closed.
